tests/notebooks/repeated_interrupt.ipynb

```diff
@@ -0,0 +1,392 @@
+      "1a2d6932cec244e88b62fecb51712565",
+      "ad34b15f3a1b44bfa95444940072d4d8",
+      "40b5fb32d97940bea5b3176d3ed2b03f",
+      "b4c3575c7aac4b36a82f34727f27870c",
+      "4075c102c94a41a2b8c43d8bf8f3b747",
+      "7fd575cab41d4ae3996dd8df3eb587ec"
+     ]
+    },
+    "id": "w17b2U_PGvXV",
+    "outputId": "08f8e107-537a-473c-c96a-857fcef7f09f"
+   },
+   "source": [
+    "while True:\n",
+    "    time.sleep(1.)"
+   ],
+   "execution_count": null,
+   "outputs": [
+    {
+     "output_type": "display_data",
+     "data": {
+      "text/html": [
+       "\n",
+       "                Tracking run with wandb version 0.10.22<br/>\n",
+       "                Syncing run <strong style=\"color:#cdcd00\">astral-snow-3</strong> to <a href=\"https://wandb.ai\" target=\"_blank\">Weights & Biases</a> <a href=\"https://docs.wandb.com/integrations/jupyter.html\" target=\"_blank\">(Documentation)</a>.<br/>\n",
+       "                Project page: <a href=\"https://wandb.ai/stas_sl/hang\" target=\"_blank\">https://wandb.ai/stas_sl/hang</a><br/>\n",
+       "                Run page: <a href=\"https://wandb.ai/stas_sl/hang/runs/1m3w4opc\" target=\"_blank\">https://wandb.ai/stas_sl/hang/runs/1m3w4opc</a><br/>\n",
+       "                Run data is saved locally in <code>/content/wandb/run-20210312_155010-1m3w4opc</code><br/><br/>\n",
+       "            "
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {
+      "tags": []
+     }
+    },
+    {
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "  | Name | Type | Params\n",
+      "------------------------------\n",
+      "------------------------------\n",
+      "100       Trainable params\n",
+      "0         Non-trainable params\n",
+      "100       Total params\n",
+      "0.000     Total estimated model params size (MB)\n"
+     ],
+     "name": "stderr"
+    },
+    {
+     "output_type": "display_data",
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "cbac275fc8ea4e27947f911dacb36a2e",
+       "version_minor": 0,
+       "version_major": 2
+      },
+      "text/plain": [
+       "HBox(children=(FloatProgress(value=1.0, bar_style='info', description='Training', layout=Layout(flex='2'), max…"
+      ]
+     },
+     "metadata": {
+      "tags": []
+     }
+    },
+    {
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ],
+     "name": "stdout"
+    },
+    {
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python3.7/dist-packages/pytorch_lightning/utilities/distributed.py:50: UserWarning: Detected KeyboardInterrupt, attempting graceful shutdown...\n",
+      "  warnings.warn(*args, **kwargs)\n"
+     ],
+     "name": "stderr"
+    },
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "1"
+      ]
+     },
+     "metadata": {
+      "tags": []
+     },
+     "execution_count": 5
```

Comment:
Why is this 5? Does it even matter?

Reply:
doesn't matter, this is just cached from me running it